packages/plugin-credentials/src/credentials/common.js

```diff
@@ -357,16 +377,16 @@ export default {
    */
   set(attrs, options) {
     if (isObject(attrs)) {
+      if (attrs.authorization) {
+        attrs = attrs.authorization;
+      }
+
       if (attrs.access_token) {
```

Comment:
I don't know what 'authorization' looks like but will it contain 'access_token' like 'attrs' possible did in the past?